endpoints.json

```diff
@@ -77,5 +81,20 @@
       "votes": 0,
       "created_at": "2024-11-25T16:25:18.024Z"
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "updates the votes of an article by the article's ID and responds with the updated article",
+    "exampleInput": { "inc_votes": 5 },
+    "exampleResponse":     {
+      "article_id": 1,
+      "title": "Living in the shadow of a great man",
+      "topic": "mitch",
+      "author": "butter_bridge",
+      "body": "I find this existence challenging",
+      "created_at": "2020-07-09T18:11:00.000Z",
+      "votes": 97,
+      "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+    }
```

Comment:
Nice, but instead of `exampleInput` I would recommend using something more similar to `exampleBody`